File: src/server/BuildingBlock/MovieTicket.Domain/Entities/Service.cs
using MovieTicket.Core.Domain;

namespace MovieTicket.Domain.Entities;

public class Service : EntityBase
{
    public String Name { get; set; }
    public String Unit { get; set; }
    public Double PriceUnit { get; set; }
    public String Avatar { get; set; }
    
    public virtual ICollection<ServiceReservation> ServiceReservations { get; set; }
}

File: src/client/mobile/app/src/main/java/com/superman/movieticket/domain/entities/Service.kt
package com.superman.movieticket.domain.entities

import com.superman.movieticket.core.entity.BaseEntity
import java.io.Serializable

data class Service (
    var name: String,
    var unit: String,
    var priceUnit: Int,
    var avatar: String,
) : BaseEntity(), Serializable